app/Controllers/configureController.php

```diff
@@ -270,7 +270,7 @@ public function archivingAction(): void {
 			];
 		if (!empty(FreshRSS_Context::userConf()->archiving['keep_period'])) {
 			$keepPeriod = FreshRSS_Context::userConf()->archiving['keep_period'];
-			if (preg_match('/^PT?(?P<count>\d+)[YMWDH]$/', $keepPeriod, $matches)) {
+			if (preg_match('/^PT?(?P<count>\d+)[YMWDH]$/', (string)$keepPeriod, $matches)) {
```

Comment:
I would better like to address the typing aspects in another PR, as I believe there are better ways than manually casting to `(string)`, so please no change of this type in this PR